Dangerfile

```diff
@@ -9,6 +9,26 @@
   end
 end
 
+# Ensure a clean commits history
 if commits.any? { |c| c.message =~ /^Merge branch '#{branch_for_merge}'/ }
   fail("Please rebase to get rid of the merge commits in this PR")
 end
+
+# Request a CHANGELOG entry, and give an example
+declared_trivial = pr_title.include?("#trivial") || pr_body.include?("#trivial")
```

Comment:
I can't stand the trivial thing. The team should decide what is trivial, rather than the PR author